src/overlays/actors/ovl_En_Door/z_en_door.c

```diff
@@ -195,6 +194,7 @@ void EnDoor_SetupType(EnDoor* this, PlayState* play) {
                 this->actionFunc = EnDoor_AjarWait;
                 this->actor.world.rot.y = -0x1800;
             }
+        fake_label:;
```

Comment:
https://decomp.me/scratch/cPdOQ

Reply:
can be matched with a scoped temp: https://decomp.me/scratch/okTFM

Reply:
Thanks! made it a player temp